### PR TITLE
working towards sending EADs to SVN

### PR DESCRIPTION
**Pull request by regineheberlein**

### CODE REVIEW

File: app/models/aspace_svn/get_eads_job.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    def handle(data_set:)
+      aspace_login
+      repos = (3..12).to_a
+      repos.each do |repo|
+
+        # make directories if they don't already exist
+        dir =
+          case repo
+            when 3
+              FileUtils.mkdir_p("mudd/publicpolicy") unless Dir.exists?("mudd/publicpolicy")
```
Comment (sandbergja) [inline app/models/aspace_svn/get_eads_job.rb:34]: One possible simplification: would it behave the same if you remove all `FileUtils` and `Dir` lines from the `case` statement, and then just add the following after the `end` of the `case` statement?

```
FileUtils.mkdir_p(dir) unless Dir.exists?(dir)
```

Comment (regineheberlein) [reply app/models/aspace_svn/get_eads_job.rb:34]: It does! Much cleaner. Thanks @sandbergja !

File: app/models/aspace_svn/get_eads_job.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        10 => "selectors",
+        11 => "ga",
+        12 => "ea"
+      }
+    end
+
+    def write_eads_to_file(dir, repo, id)
+      record = @client.get("/repositories/#{repo}/resource_descriptions/#{id}.xml")
+      ead = Nokogiri::XML(record.body)
+      # i shouldn't have to do this but I do
```
Comment (christinach) [inline app/models/aspace_svn/get_eads_job.rb:78]: Can we remove the comment?

File: app/models/aspace_svn/get_eads_job.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      ead.remove_namespaces!
+      eadid = ead.at_xpath('//eadid/text()')
+      # save file with eadid as file name
```
Comment (christinach) [inline app/models/aspace_svn/get_eads_job.rb:81]: Can we remove the comment?

File: app/models/aspace_svn/get_eads_job.rb [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      file =  File.open("#{dir}/#{eadid}.EAD.xml", "w")
+      # add back a default namespace
```
Comment (christinach) [inline app/models/aspace_svn/get_eads_job.rb:83]: Can we remove the comment?

**Review by christinach — CHANGES_REQUESTED** [review]
@regineheberlein @sandbergja thank you! I have a request regarding some comments in the code.

**Review by christinach — APPROVED** [review]
Thank you!

**Pull request merged**